### ISSUE TITLE
Elasticsearch reporting removed from gor

### ISSUE THREAD

**Issue by joekiller** [probelabs/goreplay#106]
I noticed that in #103 elasticsearch reporting was removed from gor.  I found the output immensely useful for verifying the quality of requests being sent to the replay target and how the replay target was performing. Did the external dependencies being unwieldy or was the extraction a quick fix? It it was a quick fix, I would almost suggest to those having difficulty building the binary to use the binaries already built and perhaps let the project fix the dependencies.

TLDR; I can do without the output but found it very useful and am weighing maintaining a fork just to have it back.
